docs/src/main/paradox/serialization.md

```diff
@@ -0,0 +1,75 @@
+# Serialization
+
+The general recommendation for de-/serialization of messages is to use byte arrays as value and do the de-/serialization in a `map` operation in the Akka Stream instead of implementing it directly in Kafka de-/serializers.
+
+
+## Avro with Schema Registry
+
+If you want to use [Confluent's Schema Registry](https://docs.confluent.io/current/schema-registry/docs/index.html), you need to include the dependency on `kafka-avro-serializer` as shown below. It is not available from Maven Central, that's why Confluent's repository has to be specified. These examples use `kafka-avro-seriazlizer` version $confluent.version$.
```

Comment:
As this paragraph follows the general recommendation to use serialization in `map` operators, the implicit understanding is that this paragraph is about using avro inside `map` operator.

We should add a contrasting paragraph to the general recommendation that it is possible to specify (de)serializers to in Producer/Consumer settings and here how that can be used with Avro.